test/PowerNotifications/manual-validation.md

```diff
@@ -0,0 +1,30 @@
+﻿## Name
+PowerNotifications.APITests & FunctionalTests
+
+## Feature Area
+PowerStateNotifications
+
+## Area Path
+OS\WDX\DXP\APT-Application Platform and Tools\EMR-Enterprise MSIX Runtime
+
+## ETA: < 30 mins (including setup)
+
+## Special notes
+A battery level aware VM will fail on the battery level and power source tests as the tests are currently configured for AC power devices.
+Repeat for all architectures
+
+## Test steps
+1. Install Visual Studio 2019 or later
```

Comment:
Is VS needed to run the test?
Can't this be done via the command line? TE.EXE something something something ?